packages/web-components/package.json

```diff
@@ -18,22 +18,24 @@
   "main": "dist/esm/index.js",
   "types": "dist/web-components.d.ts",
   "unpkg": "dist/web-components.min.js",
+  "beachball": {
+    "disallowedChangeTypes": []
```

Comment:
Guessing you'll be quickly following up to remove this to prevent accidental major changes, right?

Reply:
Correct - we need it to publish though and removing this now will prevent that